README.md

```diff
@@ -46,6 +46,21 @@ This image has two tags - a `latest` tag (the default), based on the most recent
 
 The web UI can be enabled by not passing any `+exec` config to the FXServer binary. This can be achieved by setting the `NO_DEFAULT_CONFIG` environment variable (see below).
 
+`txAdmin` stores it's configuration and database data in `/txAdmin`, so a volume can be set up to persist this data:
```

Comment:
The path here doesn't match the compose file below - it should be `/txData` I think?